selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -68,15 +59,17 @@ def search_and_add_user(self, user_id, role):
         # Enter user to search on
         self.find_element(*Locators.ADD_PEOPLE_SEARCH_TXT).send_keys(user_id)
         # Select role
-        self.select_role_type(role)
+        self.select_role_type(canvas_role)
 
         # Click 'Add to course' course button
         self.find_element(*Locators.ADD_TO_COURSE_BUTTON).click()
+        WebDriverWait(self._driver, 30).until_not(lambda s: s.find_element(
```

Comment:
I think this will throw a `TimeoutException` and we won't have to wait until the next assert, in fact. http://selenium-python.readthedocs.org/waits.html